src/core/lib/Modhex.mjs

```diff
@@ -0,0 +1,153 @@
+/**
+ * Modhex functions.
+ *
+ * @author linuxgemini [ilteris@asenkron.com.tr]
+ * @copyright Crown Copyright 2020
+ * @license Apache-2.0
+ */
+
+import Utils from "../Utils.mjs";
+import { toHex, fromHex } from "./Hex.mjs";
+
+/**
+ * Convert a byte array into a modhex string.
+ *
+ * @param {byteArray|Uint8Array|ArrayBuffer} data
+ * @param {string} [delim=" "]
+ * @param {number} [padding=2]
+ * @returns {string}
+ *
+ * @example
+ * // returns "cl bf bu"
+ * toModhex([10,20,30]);
+ *
+ * // returns "cl:bf:bu"
+ * toModhex([10,20,30], ":");
+ */
+export function toModhex(data, delim=" ", padding=2, extraDelim="", lineSize=0) {
```

Comment:
Generally we only use lib/* when multiple operations share the same constants or functions. I'd recommend just moving these functions into the operations files and only keeping the constants used by both operations.

Reply:
Done.